<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListWidget" name="listOutput"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtCmd"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="shortcut">
         <string>Enter, Return</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtCmd</tabstop>
  <tabstop>btnSend</tabstop>
  <tabstop>listOutput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
